ipmininet/router/config/templates/bgpd.mako

```diff
@@ -12,17 +12,28 @@ debug bgp section
 router bgp ${node.bgpd.asn}
 % if node.bgpd.routerid:
     bgp router-id ${node.bgpd.routerid}
-% elif node.ospfd:
-    bgp router-id ${node.ospfd.router_id}
+% else:
+    bgp router-id ${node.zebra.routerid}
```

Comment:
This kind of logic should happen in the build() method of the daemons, i.e. check if self.options has a routerid set or fall back to the one assigned by super()